src/inc/clrnt.h

```diff
@@ -789,6 +789,8 @@ typedef struct _DYNAMIC_FUNCTION_TABLE {
 //
 #ifdef _TARGET_AMD64_
 
+#define COMPACT_RUNTIME_FUNCTION        0
```

Comment:
The name is confusing to me. And it also seems we don't need the EndAddress for x86, or do we? If not, we can just revert the ifdef around the EndAddress to _TARGET_AMD64_ and remove this define.